### ISSUE TITLE
The AuthorizationPolicy error named: 'epPolicy:X' was not found.

### ISSUE THREAD

**Issue by Reikooters** [FastEndpoints/FastEndpoints#478]
Hi there,

After updating from 5.16.0 to 5.17.0, I'm getting the following error 500 on all endpoints.
```csharp
fail: MyProject.ExceptionHandler[0]
      =================================
      HTTP: GET /user/getUser
      TYPE: InvalidOperationException
      REASON: The AuthorizationPolicy named: 'epPolicy:MyProject.Features.MyEndpoint' was not found.
      TRACEID: 0HMTD26D1NS5J:00000002
      ---------------------------------
      Microsoft.AspNetCore.Authorization

      System.InvalidOperationException: The AuthorizationPolicy named: 'epPolicy:MyProject.Features.MyEndpoint' was not found.

         at Microsoft.AspNetCore.Authorization.AuthorizationPolicy.CombineAsync(IAuthorizationPolicyProvider policyProvider, IEnumerable`1 authorizeData, IEnumerable`1 policies)
         at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
         at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
         at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddlewareImpl.<Invoke>g__Awaited|8_0(ExceptionHandlerMiddlewareImpl middleware, HttpContext context, Task task)
```

Downgrading to 5.16.0 again resolves the issue.

For authorization, I'm using policies which look like this:
```csharp
// Authorization Policies
builder.Services.AddAuthorization(options =>
{
    options.AddPolicy("User", policy =>
    {
        policy.RequireAuthenticatedUser();

        // Check user role
        policy.RequireAssertion(c =>
        {
            return c.User.IsInRole("User") || c.User.IsInRole("Admin");
        });
    });

    options.AddPolicy("Admin", policy =>
    {
        policy.RequireAuthenticatedUser();

        // Check user role
        policy.RequireAssertion(c =>
        {
            return c.User.IsInRole("Admin");
        });
    });
});
```
And on the endpoint's Configure() method:
```csharp
Policies("User");
```
Of the two projects I've upgraded, both have the issue. One project is using the `Jwt-Or-Cookie` authentication example [taken from the FastEndpoints documentation](https://fast-endpoints.com/docs/security#combined-authentication-scheme), the other is using the built in .NET `AddJwtBearer`.

I did check the documentation in case anything had changed and I needed to update my project, but everything seemed the same as far as I could tell.

**Comment by dj-nitehawk** [comment]
one other person has reported the same issue [on discord](https://discord.com/channels/933662816458645504/1148148084484886661).

we need to know how exactly to replicate it.

could you pls provide a fresh repro project with just 1 endpoint?

i believe the most important part of the puzzle would be the order of middleware in your startup/program.cs

**Comment by dj-nitehawk** [comment]
issue fixed in `v5.17.0.1-beta`

thanks for reporting
